networkit/cpp/flow/EdmondsKarp.cpp

```diff
@@ -118,6 +118,15 @@ std::vector<node> EdmondsKarp::getSourceSet() const {
                 visited[v] = true;
             }
         });
+        if (graph->isDirected()) {
+            // Follow backwards edges in residual network
+            graph->forInNeighborsOf(u, [&](node, node v, edgeweight weight, edgeid eid) {
```

Comment:
I think you can use the signature `graph->forInNeighborsOf(u, [&](node, node v, edgeid eid)` here.

Reply:
Indeed, weight is unused. Done.